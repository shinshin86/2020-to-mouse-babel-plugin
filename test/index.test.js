const plugin = require("../lib/index");
const pluginTester = require("babel-plugin-tester").default;

pluginTester({
  plugin,
  pluginName: "2020-to-mouse-babel-plugin",
  snapshot: true,
  tests: [
    {
      title: "2020-to-mouse-babel-plugin",
      code: 'console.log("Hello 2020!");',
    },
  ],
});
