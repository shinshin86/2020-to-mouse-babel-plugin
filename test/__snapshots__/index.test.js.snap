// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`2020-to-mouse-babel-plugin 2020-to-mouse-babel-plugin: 2020-to-mouse-babel-plugin 1`] = `

console.log("Hello 2020!");

      ↓ ↓ ↓ ↓ ↓ ↓

console.log("Hello \\uD83D\\uDC2D\\uD83D\\uDC2D\\uD83D\\uDC2D\\uD83D\\uDC2D!");


`;
